src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp

```diff
@@ -669,9 +669,10 @@ namespace NativeMsh
     static PCSTR trustedAssemblies[] =
     {
         "Microsoft.ApplicationInsights",
-        "Microsoft.CodeAnalysis",
         "Microsoft.CodeAnalysis.CSharp",
+        "Microsoft.CodeAnalysis",
```

Comment:
Why was the order changed?

Reply:
The list was generated rather than trying to insert a large number of new assemblies into the list.  `Sort-Object` seems to prefer the period over end of string.